app/main.py

```diff
@@ -1,2 +1,48 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 2 / 3
+    ) -> None:
+        self._size = 0
+        self._capacity = initial_capacity
+        self._load_factor = load_factor
+        self._table = [[] for _ in range(self._capacity)]
+
+    def __setitem__(self, key: Hashable, value: any) -> None:
```

Comment:
Since `Any` from `typing` is not deprecated, you could use it in the type annotation instead of `any`